pinot-core/src/main/java/org/apache/pinot/core/segment/processing/aggregator/ValueAggregatorFactory.java

```diff
@@ -40,6 +40,10 @@ public static ValueAggregator getValueAggregator(AggregationFunctionType aggrega
         return new MaxValueAggregator(dataType);
       case SUM:
         return new SumValueAggregator(dataType);
+      case DISTINCTCOUNTRAWHLL:
```

Comment:
We can use the aggregator for both raw and non-raw function

Reply:
does that make sense? I thought these aggregators were for rolling up values in a column without changing types, so eg. `BYTES` -> `BYTES`, is it possible to change the column type here?

Reply:
does that make sense? can a column change from eg. `STRING` to `BYTES` when rolling up? or `STRING` to `LONG` when doing a distinct count?

max/min/sum are the other ones in this package and they don't change the type

Reply:
The stored value has to be serialized bytes, but we can still apply `DISTINCTCOUNTHLL` on serialized HLLs. The difference between raw and non-raw is only the final return result, where the raw one returns the HLL, the non-raw returns the count

Reply:
makes sense, either could cause this rollup on a `BYTES` column 👍

Reply:
done